src/Components/HamburgurMenu/HamburgurMenu.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import "./HamburgurMenu.css";
+import { Link } from "react-router-dom";
+import { HiHome } from "react-icons/hi";
+import { FaUserAlt } from "react-icons/fa";
+import { MdVideoLibrary, MdOutlinePlaylistPlay } from "react-icons/md";
+import { AiFillLike } from "react-icons/ai";
+import { IoIosInformationCircle } from "react-icons/io";
+import { RiPlayList2Fill } from "react-icons/ri";
+import { BsFillBookmarkFill } from "react-icons/bs";
+export const HamburgurMenuList = () => {
```

Comment:
I will suggest you to either use one single library for icons or import all icons in a separate folder and use it from there.

Reply:
nice idea, thanks tushar.